components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java

```diff
@@ -2342,6 +2368,7 @@ public static JWT encryptJWT(JWTClaimsSet jwtClaimsSet, JWEAlgorithm encryptionA
      * @return encrypted JWT token
      * @throws IdentityOAuth2Exception
      */
+    @Deprecated
```

Comment:
let's add a deprecated java doc comment as well and point to the new method

Reply:
fixed in 4d887a5